data/core/units/monsters/Ant_Egg.cfg

```diff
@@ -38,7 +43,9 @@
     advances_to=Giant Ant
     cost=4
     usage=fighter
-    description= _ "Giant Ants are common in almost any environment, and the eggs from which they emerge can often be found in caves or any sheltered environment.  Most eggs come from one of the colony queens, but the workers transport them to any suitable location."
+    description= _ "Giant Ants are common in almost any environment, and the eggs from which they emerge can often be found in caves or any sheltered environment.  Most eggs come from one of the colony queens, but the workers transport them to any suitable location.
+
+" + {ANT_EGG_TEXT}
```

Comment:
I have a feeling that maybe the paragraph break should be separated out as a non-translatable string? Something like this:

```
...to any suitable location" + "

" + {ANT_EGG_TEXT}
```

Reply:
Why is that?

Reply:
I'm not quite sure where I got this from, but I think there was a recommendation not to put whitespace at the beginning or end of translatable strings. I might be imagining it though.

Reply:
Let's see if anyone complains.

Reply:
Or if the paragraphs get stuck together into a single paragraph in some languages.